pkg/networkserver/downlink.go

```diff
@@ -142,10 +147,11 @@ func (ns *NetworkServer) generateDownlink(ctx context.Context, dev *ttnpb.EndDev
 	cmdBuf := make([]byte, 0, maxDownLen)
 	for _, cmd := range cmds {
 		var err error
+		logger.WithField("cid", cmd.CID).Debug("Adding MAC command to buffer...")
```

Comment:
This is going to generate multiple log messages for each LoRaWAN message. Are we sure we want this?